src/reason-parser/reason_lexer.mll

```diff
@@ -587,7 +608,7 @@ rule token = parse
   *)
   | "}"  { RBRACE }
   | ">}" { GREATERRBRACE }
-  | "=<" {
+  | "=<" is_jsx_not_infix_operator  {
```

Comment:
What about the infix identifier ` =</>` ?

`/` is not white space, not a newline and not a `)`, so won't this rule incorrectly fire, and then back up the lexer?

Would it be simpler to have a rule like `"=<" uppercase_or_lowercase+` that backs up the lexer? We have a similar rule for:

```
"></" uppercase_or_lowercase+
```